node/service/src/chain_spec/frequency_westend.rs

```diff
@@ -0,0 +1,40 @@
+#![allow(missing_docs, clippy::expect_used)]
+use common_runtime::constants::{FREQUENCY_TESTNET_TOKEN, TOKEN_DECIMALS};
+use frequency_runtime::Ss58Prefix;
+use polkadot_service::chain_spec::Extensions as RelayChainExtensions;
+use sc_service::ChainType;
+use sc_telemetry::TelemetryEndpoints;
+
+use super::{get_properties, Extensions};
+
+/// Specialized `ChainSpec` for the normal parachain runtime.
+pub type ChainSpec = sc_service::GenericChainSpec<Extensions>;
+
+// Generic chain spec, in case when we don't have the native runtime.
+pub type RelayChainSpec = sc_service::GenericChainSpec<RelayChainExtensions>;
+
+/// Generates the Frequency Westend chain spec
+pub fn load_frequency_westend_spec() -> ChainSpec {
+	// Give your base currency a unit name and decimal places
+	let properties =
+		get_properties(FREQUENCY_TESTNET_TOKEN, TOKEN_DECIMALS as u32, Ss58Prefix::get().into());
```

Comment:
I'm wondering if it makes sense for Westend to have a different token than Paseo? Like maybe instead of `XRQCY`, `WRQCY`?